.markdownlint.json

```diff
@@ -1,20 +1,20 @@
 {
   "default": false,
   "resultVersion": 1,
-  "MD001": true,
+  "MD001": false,
```

Comment:
Do we want to keep these disabled globally instead of using `<!-- markdownlint-disable MDXYZ-->` in the affected files?

Reply:
For Authentication API Explorer, yes. The errors I'm getting are because the tech is old and we don't have the resources or people to yet to update the experience completely. Once the Explorer is merged, I plan to go back and update this file ^

Reply:
I understand! Let's keep it like this, then